examples/angular/src/app/pages/components/footer/footer.component.ts

```diff
@@ -1,12 +1,61 @@
-import { CommonModule } from '@angular/common';
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { environment } from '../../../../environments/environment';
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+
+import { DOTCMS_CLIENT_TOKEN } from '../../../client-token/dotcms-client';
+import { BlogsComponent } from '../../../components/blogs/blogs.component';
+import { DestinationsComponent } from '../../../components/destinations/destinations.component';
 
 @Component({
   selector: 'app-footer',
   standalone: true,
-  imports: [],
-  template: `<footer class="p-4 text-white bg-red-400">Footer</footer>`,
+  imports: [BlogsComponent, DestinationsComponent],
+  template: `<footer class="p-4 text-white bg-red-100 py-24">
+    <div
+      class="grid md:grid-cols-3 sm:grid-cols-1 md:grid-rows-1 sm:grid-rows-3 gap-7 mx-24"
+    >
+      <div class="flex gap-7 flex-col">
+        <h2 class="text-2xl font-bold text-black">About us</h2>
+        <p class="text-sm text-zinc-800">
+          We are TravelLux, a community of dedicated travel experts,
+          journalists, and bloggers. Our aim is to offer you the best insight on
+          where to go for your travel as well as to give you amazing
+          opportunities with free benefits and bonuses for registered clients.
+        </p>
+        <img />
+        <img
+          [src]="
```

Comment:
@zJaaal Should we encourage customers to use ngSrc?

Reply:
I agree, done!